bots/bot.go

```diff
@@ -0,0 +1,36 @@
+package bot
+
+import "crypto/tls"
+
+type BotHandleFunc func(string, []byte)
+
+type Bot interface {
+	HandleRegistration(string, []byte) string
+	Run()
+	Stop()
+}
+
+func SendRequestToCONIKS(address string, msg []byte) ([]byte, error) {
+	conf := &tls.Config{
+		InsecureSkipVerify: true,
```

Comment:
I would prefer to either use no TLS connection between the bot and the coniks server (as long as they run on the same machine) or to not use the `InsecureSkipVerify` mode (see https://golang.org/src/crypto/tls/common.go?s=#L302 for details)